app.py

```diff
@@ -1,12 +1,22 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+from flask import request
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/add_Item', methods=['POST'])
+def add_Item():
+    item = request.form['title']
+    session.add_item(item)
+
+    items = session.get_items()
+    return render_template('index.html', items = items)
```

Comment:
Instead of these two lines, you can `return redirect('/')`
This means the browser will return a redirect response sending the user to the index page. 

This means they see the correct url, refreshing the page works correctly and also you have less duplicated code.